### ISSUE TITLE
Why you are making a module under our brand WEB4PRO?

### ISSUE THREAD

**Issue by sfomenko** [sivajik34/Magento2-Product-Attachments#4]
Hi Sivakumar,

We are interested why are you using our brand making this extension?

**Comment by sivajik34** [comment]
Yes , I have taken m1 extension.really sorry about that.
I will remove from Github if its a concern for you.

**Comment by sfomenko** [comment]
Yes, please. We don't want to have any mentions of our company in third-party developments.
Also, we'll release our own Magento2 Product Attachments on marketplace shortly and we wouldn't like to face misunderstandings.

Thanks.
